src/pretix/static/pretixcontrol/js/ui/main.js

```diff
@@ -411,8 +414,10 @@ var form_handlers = function (el) {
                 }
             };
         update();
-        dependency.closest('.form-group').find('[name=' + dependency.attr("name") + ']').on("change", update);
-        dependency.closest('.form-group').find('[name=' + dependency.attr("name") + ']').on("dp.change", update);
+        dependency.each(function() {
+            $(this).closest('.form-group').find('[name=' + $(this).attr("name") + ']').on("change", update);
+            $(this).closest('.form-group').find('[name=' + $(this).attr("name") + ']').on("dp.change", update);
```

Comment:
As it is the same element, it can be compressed to:
```suggestion
            $(this).closest('.form-group').find('[name=' + $(this).attr("name") + ']').on("change dp.change", update);
```